htdocs/admin/multicurrency.php

```diff
@@ -41,10 +42,17 @@
 
 // Parameters
 $action = GETPOST('action', 'alpha');
-
+$sync_response = GETPOST('sync_response');
 /*
  * Actions
  */
+if (!empty($sync_response))
+{
+	$sync_response = json_decode($sync_response);
+	MultiCurrency::syncRates($sync_response);
+	exit;
```

Comment:
No, the synchronize work from JS, if we got error from the webservice I notice it by JS event or if all is right I refresh by JS